cylc/flow/cfgspec/workflow.py

```diff
@@ -1299,14 +1311,31 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                 job scripts to run.
 
             ''') + REPLACED_BY_PLATFORMS):
-                Conf('batch system', VDR.V_STRING)
-                Conf('batch submit command template', VDR.V_STRING)
+                Conf('batch system', VDR.V_STRING, desc='''
+                    Batch/Queuing system to submit task jobs to.
+
+                    .. seealso::
+
+                       Kept for back compatibility but replaced by
+                       :cylc:conf:`global.cylc[platforms][<platform name>]
+                       job runner`.
```

Comment:
Should this be a deprecation notice?

```
.. deprecated:: 8.0.0

   Replaced by :ref:`platforms <...>`.
```

(the configuration is not parsec.deprecated, however, the feature is deprecated in that we want people to move away from it so we can remove it in the near future)

Reply:
Should be same as REPLACED_BY_PLATFORMS?

Reply:
> Should be same as REPLACED_BY_PLATFORMS?

No, because it's not only been removed, but renamed too.